hil/ext/switches/dell.py

```diff
@@ -55,6 +57,18 @@ def validate(kwargs):
     def session(self):
         return _PowerConnect55xxSession.connect(self)
 
+    @staticmethod
+    def validate_port_name(port):
+        """
+        Valid port names for this switch are of the form gi1/0/11,
+        te1/0/12, gi1/12, or te1/3
+        """
+
+        val = re.compile('(^(gi|te)\d+[/]\d+[/]\d+$)|(^(gi|te)\d+[/]\d+$)')
```

Comment:
Thanks for the regex help. I was trying to do it with the `|` operator, but that didn't work for some reason. Will try your suggestion.